pipeline-templates/package-vsix.yaml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  pool: ''
+
+jobs:
+- job: ${{ parameters.pool.os }}_Package
+  pool:
+    ${{ if eq(parameters.oes, 'true') }}:
+        ${{ parameters.pool }}
+    ${{ else }}:
+        vmImage: ${{ parameters.pool.vmImage }}
+  displayName: '📦 Package and Publish'
+  dependsOn:
+  - ${{ parameters.pool.os }}_Build
+  - TSLint
+  condition: succeeded()

```

Comment:
nit: succeeded is default

Reply:
I'm not sure I trust the yaml team to not write a bug in the future that breaks this 😂 

I could not find any documentation from them stating that it is the intended behavior. https://learn.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml%2Cstages. So while I think this is good feedback to remove unnecessary code, I would prefer to keep it as a safety measure.